spack.yaml

```diff
@@ -4,6 +4,7 @@ spack:
   - "pil@2.9:"
   - "python@3.6:+bz2"
   - "py-black@21:"
+  - "py-einops"
```

Comment:
Looks like we don't yet have a Spack package for `py-einops`, I'll add one.

Reply:
My fault, I assumed there was one.